internal/server/server.go

```diff
@@ -21,6 +23,60 @@ import (
 
 type Handlers struct{}
 
+func OsbuildClient() (*cloudapi.Client, error) {
+	socket, ok := os.LookupEnv("OSBUILD_SERVICE")
```

Comment:
I think we can rely on this being set, and it being `https`, no need to support anything else, I think (not sure composer even does).

Reply:
i made it not mandatory to be https for now, that way I can figure out how to put all the certs/secrets in place on clouddot stage without fiddling on this PR.